packages/sakuli-legacy/src/init-command/create-structure.ts

```diff
@@ -0,0 +1,70 @@
+import { mkdirSync, writeFileSync } from "fs";
+import { stripIndent } from 'common-tags';
+
+export const createTestsuite = (directory: string, suiteName: string) => {
+    const rootDir = directory + `/${suiteName}`;
+    mkdirSync(rootDir, {recursive: true});
+    console.log(`Created directory ${suiteName}`);
+    createTestsuiteProperties(rootDir, suiteName);
+    createTestsuiteSuite(rootDir);
+    createTestcaseInTestsuite(rootDir);
+};
+
+export const createPackageJson = (path: string, suiteName: string) => {
+    const packageJson = {
+        name: suiteName,
+        version: "1.0.0",
+        description: "",
+        main: "index.js",
+        scripts: {
+            test: `sakuli run ${suiteName}`
+        },
+        author: "",
+        license: "ISC",
+        dependencies: {
+            "@sakuli/cli": "^2.1.3",
+        },
+    };
+
+    writeFileSync(`${path}/package.json`, JSON.stringify(packageJson, null, 4));
+    console.log("Created file package.json")
+};
+
+const createTestsuiteProperties = (path: string, suiteName: string) => {
+    writeFileSync(
+        `${path}/testsuite.properties`,
+        stripIndent`
+        testsuite.id=${suiteName}
+        testsuite.browser=chrome
+        #testsuite.name=\${testsuite.id}
+        #testsuite.warningTime=0
+        #testsuite.criticalTime=0
+        #testsuite.uiOnly=false
+        
+        #log.level=INFO
+        #sakuli.log.maxAge=14
+        #sakuli.log.folder=\${sakuli.testsuite.folder}/_logs
+        #sakuli.environment.similarity.default=0.99
+        #sakuli.autoHighlight=false
+        #sakuli.highlight.seconds=0.2
+        #sakuli.typeDelay=300
+        #sakuli.clickDelay=0.2
+        #sakuli.encryption.key=""
+        #sakuli.screenshot.onError=true
+        #sakuli.screenshot.dir=\${sakuli.log.folder}/_screenshots
```

Comment:
contents of generated files could be externalized into a "template" file or a single module which expose it as a constant.